client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -280,61 +280,29 @@ export class DisplayTable extends React.Component {
       .reverse()
       .value();
 
-    const {
-      paginate_by,
-      paginated_data,
-      change_paginate_by,
-      show_pagination_controls,
-      page_selector,
-    } = (() => {
-      const paginate_by =
-        size_to_paginate && _.size(sorted_filtered_data) > size_to_paginate
-          ? state_paginate_by
-          : 0;
-
-      const num_pages_exact = paginate_by
-        ? _.size(sorted_filtered_data) / paginate_by
-        : -1;
```

Comment:
Both of these encode future logic operation in their stored values by assigning special meaning to values which would otherwise hold no meaning (page size of 0, or a negative number of pages). I know this kind of thing goes on in the `Typeahead` to some extent, but I very strongly suggest against it! Bad for code legibility and a future-bug waiting to happen, IMO.